### ISSUE TITLE
[TF2] [Feature Request] Payload race should be a Core game mode, not Alternative

### ISSUE THREAD

**Issue by Rongeno** [ValveSoftware/Source-1-Games#4151]
So, for some weird reason, currently Payload Race is considered an alternative gamemode in casual gamemode sekect section, which is weird considering the fact that it doesn't have anything wacky or weird to be one. 
![image](https://user-images.githubusercontent.com/108897156/183887729-64b4dd3b-158d-4c3f-bbfd-bae7620401c7.png)
Because of that, official Payload Race servers don't have much people that play it, because most people don't realy bother going deep down into gamemode selection. Not only that, but new players don't even know about it's existance until playing like 30-40 hours in this game, which is sad, considering that Payload Race has 4 unique good maps.

**Comment by kisak-valve** [comment]
Possible duplicate of #2220.

**Comment by condor00fr** [comment]
I agree, I don't see why it wasn't included in the base selection, another thing i would like to see is the return of arena but as one of the alternate gamemodes, it even has a hud icon for it in the game files.

**Comment by DrCactusTF2** [comment]
Personally I think the "alternative" pool should just be gotten rid of altogether and the maps merged with the normal pool
